demo/src/app/components/+tabs/tabs-section.list.ts

```diff
@@ -45,6 +46,13 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/manual/manual.html'),
         outlet: DemoTabsManualComponent
       },
+      {
+        title: 'Disable selection',
+        anchor: 'tabs-disable',
```

Comment:
`tabs disable` and `disable selection` not the same, try not to change meaning of anchor.

Reply:
fix it